spec/controllers/api/v1/tokens_controller_spec.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe API::V1::TokensController, type: :controller do
+  describe 'POST#revoke' do
```

Comment:
Can we write a happy path test case also where the user able to log out successfully?

Reply:
Good catch! Because I override only about `revocation_error_response`, and that's is related to the failed case, then I only test about it. For the happy path test case, I think `doorkeeper` already test about it, that's why I didn't add the test for it.
I will add the happy path case for this.